addons/mail/static/tests/messaging_menu/messaging_menu_tests.js

```diff
@@ -546,6 +546,7 @@ QUnit.test("basic rendering", async (assert) => {
     await contains("button", { text: "New Message" });
     await contains(".o-mail-MessagingMenu div.text-muted", { text: "No conversation yet..." });
     await click(".o_menu_systray .dropdown-toggle:has(i[aria-label='Messages'])");
+    await nextTick();
```

Comment:
The following assert should be adapted to use contains rather than introducing an arbitrary tick.